src/algebra/category/Group/monoidal.lean

```diff
@@ -0,0 +1,142 @@
+import category_theory.monoidal.of_has_finite_products
+import category_theory.monoidal.Mon_
+import algebra.category.Group.limits
+import algebra.category.Group.biproducts
+import algebra.category.Group.abelian
+import category_theory.elementwise
+
+noncomputable theory
+
+universes u
+
+open category_theory category_theory.monoidal_category category_theory.limits
+open_locale zero_object
+
+namespace AddCommGroup
+
+namespace Mon_
+
+instance : category_theory.monoidal_category AddCommGroup.{u} :=
```

Comment:
Monoidal structure is wrong

Reply:
See `algebra/category/Module/monoidal.lean`.

Reply:
The more general statement should be that if `R` is a commutative ring, `R`-algebras identify to monoids in the category of `R`-modules.

Reply:
That more general statement is in `category_theory/module/internal/Module.lean`.

Reply:
I am aware of the results, but I want to show monoids in category of sheaves of abelian group is sheaves of rings, so I need the more specific statement that monoids in category of Ab are rings.